containers/Makefile

```diff
@@ -71,12 +75,13 @@ clean-strongswan:
 # 1. Configs StrongSwan for client
 # 2. Builds StrongSwan from source
 # 3. Builds a debian containing StrongSwan
-build-strongswan-client:
+build-strongswan-client: .BIN_FMT
 	docker run --rm \
 		-v ${strongswan_dir}/:/strongswan \
+		-v ${strongswan_dir}/../packages:/root/packages/strongswan/aarch64 \
 		-e CLIENT_ARCH=${CLIENT_ARCH} \
 		${CLIENT_BUILDER_IMAGE_NAME} \
-		bash -c '/strongswan/containers/scripts/build_strongswan.sh build-strongswan-client'
+		bash -c '/strongswan/containers/scripts/build_strongswan.sh build-strongswan-client-apk'
```

Comment:
If `build-strongswan-client` is no longer used then remove it from `build_strongswan.sh`